### PR TITLE
Add internal_index_image_copy and internal_index_image_copy_resolved …

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
…attributes

Added attributes to response of Add and Rm requests. Their values will be the
pullspec pointing to the default IIB output repo. The values will be same as
that of index_image and index_image_resolved respectively when overwrite_from_index
is NOT provided when submitting the request.

Refers to CLOUDDST-13288

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
@release-engineering/exd-guild-hello-operator Could you please review?

**Review by JAVGan — APPROVED** [review]
LGTM

File: tests/test_web/test_api_v1.py [hunk 1]
```diff
@@ -680,6 +688,8 @@ def test_add_bundle_success(
         'id': 1,
         'index_image': None,
         'index_image_resolved': None,
+        'internal_index_image_copy': None,
```
Comment (chandwanitulsi) [inline tests/test_web/test_api_v1.py:691]: Why would this be None?

**Comment by pbortlov** [comment]
@yashvardhannanavati are we adding the attributes to iiblib?

**Comment by yashvardhannanavati** [comment]
@pbortlov We will have to if we want Pub to use it.

**Comment by yashvardhannanavati** [comment]
https://github.com/release-engineering/iiblib/pull/54 iiblib PR :)

**Review by chandwanitulsi — APPROVED** [review]

**Pull request merged**